vtm-extras/src/org/oscim/tiling/source/geojson/GeoJsonTileDecoder.java

```diff
@@ -268,11 +268,6 @@ private void parsePolygon(JsonParser jp)
         }
     }
 
-    private void removeLastPoint() {
-        mMapElement.pointNextPos -= 2;
-        mMapElement.index[mMapElement.indexCurrentPos] -= 2;
-    }
-
```

Comment:
Would it be ok, to move this method to GeometryBuffer, as it is used 3 times with same implementation?

Reply:
Isn't that specific to implementation as map element can have 2D or 3D points?

Reply:
Many methods in GeomtryBuffer are specific to implementation...or could distinguish and make if clause with type before removing. If it's no `TRIS`, it's a 2D point.

Reply:
> or could distinguish and make if clause with type before removing. If it's no TRIS, it's a 2D point

To cover all options in a new method would need such check.